README.md

```diff
@@ -10,6 +10,21 @@ Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
 
 Ready to run in production? Please [check our deployment guides](https://hexdocs.pm/phoenix/deployment.html).
 
+### Generating LiveView, templates, and context
+```code iex
+mix phx.gen.live Words User users uid:uuid name:text
```

Comment:
I think it will make a little more sense when we get further into our app, but I am not sure that we want the users CRUD (create, read, update, delete) functions in the same context as the better and worst words.